client/src/components/Button.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Button({ myFunction, buttonTitle }) {
+  return <button onClick={myFunction}>{buttonTitle}</button>;
+}
+
+export default Button;
```

Comment:
I like! 👍 But no stylings for this button? 🤔

Reply:
We first took care of the functionality, and today we take care of the styling. 👍